### DISCUSSION TITLE
Change PWA URL without reinstalling the app

### DISCUSSION THREAD

**Discussion by prakash2091** [vercel/next.js#45751] [Help]
I have a NEXT JS PWA running in production installed on many of my users devices on https://url1.mydomain.com and I need to move it on https://url2.mydomain.com.

I tried to do a simple redirect using window.location.href.replace(, and it seems to work on normal browsers, but on iOS/Android installed PWA app, instead of staying full screen, the app suddenly shows a browser url and footer bar at the top and bottom of the screen as if the app was accessing the in-app browser treating the new url as external to the app.

**Comment by N-M** [comment]
Do you have .well-known/assetlinks.json on both urls?

it should have value similar to this 

```
[
  {
    "relation": ["delegate_permission/common.handle_all_urls"],
    "target": {
      "namespace": "android_app",
      "package_name": "com.dashboard",
      "sha256_cert_fingerprints":
      ["B:6A:56:67:5A:4C:A5:6C:75"]
    }
  }
]
```

**Comment by leerob** [comment]
Hey everyone. We now have updated documentation on building PWAs with Next.js.

https://nextjs.org/docs/app/building-your-application/configuring/progressive-web-apps
